README.md

```diff
@@ -1,2 +1,121 @@
-# generate-release-notes
-Efficiently automate your release note generation with 'generate-release-notes'. This tool scans your target GitHub repository's issues, sorting and organizing them into well-formatted release notes. Perfect for maintaining a streamlined and organized release process.
+# Generate Release Notes Action
+
+This GitHub Action automatically generates release notes for a given release tag by categorizing contributions into user-defined chapters based on labels. It streamlines the process of documenting new features, bug fixes, and breaking changes in your project releases.
+
+## Motivation
+
+Generate Release Notes action is dedicated to enhance the quality and organization of project documentation. Its primary purpose is to categorize release notes according to various labels, perfectly aligning with the unique organizational needs of each project. In addition, it offers robust support for acknowledging the efforts of contributors, thereby fostering a sense of recognition and appreciation within the team. Another noteworthy feature of this tool is its capability to detect potential gaps in documentation, ensuring that release notes are comprehensive and leave no stone unturned. Well maintained release notes are a vital component in maintaining high-quality, meticulously organized documentation, which is indispensable in the development process. This repository reflects our commitment to excellence in project documentation and team collaboration.
+
+## Setup and Usage
+
+### Prerequisites
+
+Before we begin, ensure you have a GitHub Token with permission to fetch repository data such as Issues and Pull Requests.
+
+### Adding the Action to Your Workflow
+
+Add the following step to your GitHub workflow:
+
+```yaml
+- name: Generate Release Notes
+  id: release_notes
+  uses: AbsaOSS/generate-release-notes@0.1.0
+  with:
+    github_token: ${{ secrets.GITHUB_TOKEN }}
+    chapters: '[
+      {"title": "Breaking Changes 💥", "label": "breaking-change"},
+      {"title": "New Features 🎉", "label": "enhancement"},
+      {"title": "Bugfixes 🛠", "label": "bugfix"}
+    ]'
+    warnings: true
+```
```

Comment:
Please add echo example of printing out to STDOUT the release notes

Reply:
Added in commit - 42931a69714b0b9afc483184c985938d9bc67113.